src/libertem/udf/sumsigudf.py

```diff
@@ -0,0 +1,21 @@
+import numpy as np
+
+from libertem.udf import UDF
+
+
+class sumsigUDF(UDF):
```

Comment:
Classes should be in `CapitalCamelCase`, so `SumsigUDF` or `SumSigUDF` here (matching the `SumSigAnalysis`).